src/mobility/components/BenefitTile.tsx

```diff
@@ -0,0 +1,149 @@
+import {ThemeText, screenReaderPause} from '@atb/components/text';
+import {TileWithButton} from '@atb/components/tile';
+import {StyleSheet} from '@atb/theme';
+import {InteractiveColor} from '@atb/theme/colors';
+import {View, ViewStyle} from 'react-native';
+import {BenefitImageAsset} from './BenefitImage';
+import {FareProductBenefitType} from '../use-operator-benefits-for-fare-product';
+import {getTextForLanguage, useTranslation} from '@atb/translations';
+import {MobilityTexts} from '@atb/translations/screens/subscreens/MobilityTexts';
+import {Map} from '@atb/assets/svg/mono-icons/map';
+import {
+  MapFilterType,
+  MobilityMapFilterType,
+  useUserMapFilters,
+} from '@atb/components/map';
+import {FormFactor} from '@atb/api/types/generated/mobility-types_v2';
+import {getNewFilterState} from '../utils';
+import {useOperators} from '../use-operators';
+
+type BenefitCardProps = {
+  interactiveColor: InteractiveColor;
+  style?: ViewStyle;
+  benefit: FareProductBenefitType;
+  onNavigateToMap?: (initialFilters: MapFilterType) => void;
+};
+
+export const BenefitTile = ({
+  style,
+  benefit,
+  onNavigateToMap,
+}: BenefitCardProps): JSX.Element => {
+  const styles = useStyles();
+  const {t, language} = useTranslation();
+  const title = t(
+    MobilityTexts.formFactor(benefit.formFactors[0] as FormFactor),
+  );
+  const description = getTextForLanguage(benefit.ticketDescription, language);
+
+  const {getMapFilter, setMapFilter} = useUserMapFilters();
+  const operators = useOperators();
+
+  const onPress = onNavigateToMap
+    ? async () => {
+        const storedFilters = await getMapFilter();
+
+        let mobilityFilters: MobilityMapFilterType = storedFilters.mobility;
+        benefit.formFactors.forEach((formFactor) => {
+          const allOperators = operators.byFormFactor(FormFactor.Scooter);
```

Comment:
May just be me missing the point, but should it be `byFormFactor(FormFactor.Scooter)` here?

Reply:
Good catch! You're not missing anything, just some temporary code I forgot to replace. Interesting that it still worked like that, tho 😄